src/ingest_validation_tools/directory-schemas/codex.yaml

```
@@ -17,50 +10,62 @@
+  pattern: '(raw|src_.*)+exposure_times\.txt'
+  description: 'Comma separated text file used for background subtraction that contains valid exposure times for all cycles [e.g: Cycle,CH1,CH2,CH3,CH4].'
   required: False
 -
-  pattern: 'src_[^/]+/exposure_times\.txt'
-  description: 'Comma separated text file used for background subtraction that contains valid exposure times for all cycles [e.g: Cycle,CH1,CH2,CH3,CH4].'
+  pattern: 'raw/reg_00\.png'
+  description: 'Region overviews'
+  required: False
+-
+  pattern: '((raw|src_.*)+/(experiment|Experiment)\.json)'
+  description: 'JSON file produced by the Akoya software which contains the metadata for the experiment, including the software version used, microscope parameters, channel names, pixel dimensions, etc. (required for HuBMAP pipeline)'
   required: True
 -
-  pattern: 'drv_[^/]+/exposure_times\.txt'
-  description: 'Comma separated text file used for background subtraction that contains valid exposure times for all cycles [e.g: Cycle,CH1,CH2,CH3,CH4].'
+  pattern: '((raw|src.*)/segmentation\.json)'
+  description: 'JSON file produced by the Akoya software with segmentation information'
   required: False
 -
-  pattern: 'src_[^/]+/segmentation\.(json|txt)'
-  description: 'JSON file produced by the Akoya software which contains the parameters used for segmentation. (required for HuBMAP pipeline)'
-  required: True
+  pattern: 'processed/HandE\.tif'
+  description: 'HandE image'
+  required: False
+-
+  pattern: 'processed/HandE_RGB\.tif'
+  description: 'HandE RGB image'
+  required: False
 -
-  pattern: 'drv_[^/]+/segmentation\.(json|txt)'
-  description: 'JSON file produced by the Akoya software which contains the parameters used for segmentation. (required for HuBMAP pipeline)'
+  pattern: 'processed/HandE_RGB_thumbnail.jpg'
+  description: 'HandE RGB thumbnail'
   required: False
 -
-  pattern: 'drv_[^/]+/processed_[^/]+/.*'
-  description: 'processed files produced by the Akoya software, not used by the HIVE'
-  ignore: True
+  pattern: '(processed|drv_.*|extras)/config\.txt'
+  description: 'Configuration file with information needed for the HuBMAP processing pipeline'
+  required: False
 -
-  pattern: 'src_[^/]+/channelnames_report\.csv'
-  description: 'Two column CSV: The first column is a name or target; The second column is boolean: "FALSE" channels are excluded from processing. (required for HuBMAP pipeline)'
+  pattern: '(raw/segmentation\.json|(processed|drv_src.*|extras)/config\.txt)'
+  description: 'Sanity check that verifies the existence of at least one of these files. This is required for the HuBMAP processing pipeline'
   required: True
 -
-  pattern: 'src_[^/]+/channelnames\.txt'
-  description: 'Text file produced by the Akoya software which contains the (linearized) channel number and the Name/ID/Target of the channel (required for HuBMAP pipeline)'
-  required: True
+  pattern: 'src_.*[^/]+/channelnames_report\.csv'
```

Why do you add the `.*`?
```suggestion
  pattern: 'src_[^/]+/channelnames_report\.csv'
```